test/addons/rbd-mirror/start

```diff
@@ -231,6 +231,15 @@ def deploy_vrc_sample(cluster):
     )
 
 
+def deploy_vrc_flatten_sample(cluster):
```

Comment:
Why are we creating a VRClass with flatten support always in drenv? Or, what is the plan with this change? Do we intend to add a test that uses/selects this VRC in the e2e runs?